po/de.po

```diff
@@ -18875,11 +18875,11 @@ msgstr "Online-Auftrag SEPA-Überweisung"
 
 #: gnucash/import-export/aqb/gnc-ab-transfer.c:218
 msgid "Online Banking European (SEPA) Internal Transfer"
-msgstr "Online-Auftrag SEPA-Übertrag"
+msgstr "Online-Auftrag interne SEPA-Umbuchung"
```

Comment:
Bank-interne Umbuchung.
Die alte, nationale Methode wird, zumindest in FinTS, vermutlich aber auch EBICs, nicht mehr unterstützt. Von daher könnte man die Einträge alle vereinfachen (erst im Quelltext, dann in den Po's)